### ISSUE TITLE
VB -> C#: CInt wrongly converted to (int)

### ISSUE THREAD

**Issue by marcal** [icsharpcode/CodeConverter#658]
Hello,

I have seen a problem with the conversion of the CInt.
Cint (99.99) gives 100
While (int) 99.99 gives 99

Marc

### Input code
```vbnet
 Dim do_Tmp As Double = 9999 / 100
        MessageBox.Show(CInt(do_Tmp))
```

### Erroneous output
```C#
double do_Tmp = 9999d / 100d;
        MessageBox.Show(((int)do_Tmp).ToString ()));
```

### Expected output
```C#


double do_Tmp = 9999d / 100d;
        MessageBox.Show((Conversions.ToInteger (do_Tmp).ToString ());
```

### Details
Last plugin version (.NET) and the online converter have the problem

**Comment by GrahamTheCoder** [comment]
Hopefully it's just a case of making sure a conversion method is found somewhere around this point:
https://github.com/icsharpcode/CodeConverter/blob/a5473fbf194d70daa6b47a5267c22337a6eef2e7/CodeConverter/CSharp/ExpressionNodeVisitor.cs#L213-L217

**Comment by GrahamTheCoder** [comment]
Fixed by #669
